Mathlib/Algebra/Order/Interval/Finset/SuccPred.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2021 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Group.Embedding
+import Mathlib.Algebra.Order.Interval.Set.Monoid
+import Mathlib.Algebra.Order.Interval.Finset.Basic
+import Mathlib.Algebra.Order.SuccPred
+import Mathlib.Order.Interval.Finset.SuccPred
+
+/-!
+# Finset intervals in a successor-predecessor order
+
+This files proves relations between the various finset intervals in a successor/predecessor order.
+-/
```

Comment:
Let's have a TODO (in this one and the other file) for the half-infinite versions of these too.